<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>679</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>679</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>679</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>下载M3U8地址的视频</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_folder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>此处粘贴输出目录</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_m3u8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="placeholderText">
     <string>此处粘贴M3U8地址，可以多条换行分开</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="handleWidth">
     <number>20</number>
    </property>
    <widget class="QPushButton" name="pushButton_start">
     <property name="text">
      <string>开始下载</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_openfolder">
     <property name="text">
      <string>打开输出目录</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>511</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionabout">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
